src/main/webapp/DastPipelineGenerator.js

```diff
@@ -37,7 +37,7 @@ const commonWebScopeSetting = closestRow('#dast-common-scope');
 const commonWebScopeSettingAttr = closestRow('#dast-common-scope-attr');
 const commonScanPolicy = closestRow('#dast-standard-scan-policy');
 const apiSettingRowEntry = closestRow('#dast-api-scan-block');
-

```

Comment:
Please check the visibility in case of No web scan type, This control should be hidden.